### PR TITLE
bump aoc collector to v0.9.0

### PR DESCRIPTION
**Pull request by wangzlei**
Contains 2 changes:
1. bump AOC collector to v0.9.0 (upstream 0.25)
2. update python LAMBDA_EXEC_WRAPPER, refer [upstream](https://github.com/open-telemetry/opentelemetry-lambda/commit/4fc7483587ff82f2a679ca308e078c833601fef8#diff-3bdb05ad1b8dfe19a9868aa352532bc42af15de7035c12c7b909a6475fd35206)

### CODE REVIEW

File: adot/python/src/otel/otel_sdk/otel-instrument [hunk 1]
```diff
@@ -2,17 +2,46 @@
...
+            environ.get("AWS_LAMBDA_FUNCTION_VERSION"),
+        )
+)
+environ["OTEL_RESOURCE_ATTRIBUTES"] = "%s,%s" % (
+    lambda_resource_attributes,
+    environ.get("OTEL_RESOURCE_ATTRIBUTES"),
+)
+
+# xray propagator
+environ["OTEL_PROPAGATORS"] = "aws_xray"
```
Comment (wangzlei) [inline adot/python/src/otel/otel_sdk/otel-instrument:44]: TODO: will have a nested script to append this change.

Comment (anuraaga) [reply adot/python/src/otel/otel_sdk/otel-instrument:44]: Hmm - this value seems against spec

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration

Maybe you can help fix in python upstream

File: adot/python/src/otel/otel_sdk/otel-instrument [hunk 2]
```diff
@@ -2,17 +2,46 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT-0
 
-import os
+from os import environ, system
```
Comment (anuraaga) [inline adot/python/src/otel/otel_sdk/otel-instrument:5]: By the way, do you expect this script to get more complicated? If only setting variables / interpolating strings, bash script would be more readable and same maintanability.

**Review by anuraaga — APPROVED** [review]

File: adot/python/src/otel/otel_sdk/otel-instrument (hunk 2 above)
Comment (wangzlei) [reply adot/python/src/otel/otel_sdk/otel-instrument:5]: Python have to setup otel component totally by environment variables, sdk does not support combined mode(env variable + code), we don't want to override customer's env variables, so have to add simple operation such as null check, append, etc.

**Comment by wangzlei** [comment]
AOC lambdaComponents republish tag v0.9.0 to fix a compiling issue. But seems even the tag can be deleted and reused, but the go module would not be override. Have to ask AOC maintainer to publish v0.9.1 to solve the compiling problem

**Comment by wangzlei** [comment]
pass build by new AOC lambdaComponents v0.9.1

**Pull request merged**